src/runtime/tests/particle-interface-loading-test.ts

```diff
@@ -84,11 +84,14 @@ describe('particle interface loading', () => {
       ],
     });
 
-    const ifaceStore = await arc.createStore(ifaceType) as SingletonStorageProvider;
-    await ifaceStore.set(manifest.particles[0].toLiteral());
+    const ifaceStore = await arc.createStore(ifaceType);
+    const ifaceHandle = await singletonHandleForTest(arc, ifaceStore);
+    await ifaceHandle.set(manifest.particles[0].toLiteral());
```

Comment:
this probably does not work with the old stack (it expects an entity), I think you can keep using the SingletonStorageProvider if flag is false

Reply:
Argh, I thought I'd checked that. Oh well, updated.